KubeArmor/core/kubeUpdate.go

```diff
@@ -2761,6 +2761,7 @@ func (dm *KubeArmorDaemon) WatchConfigMap() cache.InformerSynced {
 			if cm, ok := obj.(*corev1.ConfigMap); ok && cm.Namespace == cmNS {
 				cfg.GlobalCfg.HostVisibility = cm.Data[cfg.ConfigHostVisibility]
 				cfg.GlobalCfg.Visibility = cm.Data[cfg.ConfigVisibility]
+				cfg.GlobalCfg.Cluster = cm.Data[cfg.ConfigCluster]
```

Comment:
we need to update `dm.Node.ClusterName` as well to reflect the changes for the node.